src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -97,6 +98,16 @@ def execute(self, context):
         self.log.info(f"returning table {self.output_table}")
         return self.output_table
 
+    def _set_default_values_from_dag_default(self, context):
+        self.schema = self.schema or get_schema(context)
```

Comment:
Why don't we currently try to retrieve the schema from the `context["dag"]` as well, like the other parameters?